fleet-cmdr/src/handlers/cmd_orb_details.rs

```diff
@@ -0,0 +1,46 @@
+use async_trait::async_trait;
+use color_eyre::eyre::Result;
+use orb_relay_client::{QoS, RecvMessage};
+use orb_relay_messages::{
+    orb_commands::v1::{OrbCommandError, OrbDetailsRequest, OrbDetailsResponse},
+    prost::Message,
+};
+use tracing::info;
+
+use super::OrbCommandHandler;
+
+pub struct OrbDetailsCommandHandler {}
+
+impl OrbDetailsCommandHandler {
+    pub fn new() -> Self {
+        Self {}
+    }
+}
+
+#[async_trait]
+impl OrbCommandHandler for OrbDetailsCommandHandler {
+    async fn handle(&self, command: &RecvMessage) -> Result<(), OrbCommandError> {
+        info!("handling orb details command");
+        let _request = OrbDetailsRequest::decode(command.payload.as_slice()).unwrap();
```

Comment:
Lets punt this implementation to a subsequent pr, using emptry strings are fine for now.
```suggestion
        let _request = OrbDetailsRequest::decode(command.payload.as_slice()).unwrap();
        // TODO(paulquinn00): Consult with @oldgalileo and @sfikastheo to determine where to get this info from.
```